service/src/main/resources/bootstrap.yaml

```diff
@@ -14,5 +14,6 @@ spring:
         local-authority-code-to-hmcts-court-mapping: fpl.local_authority_code_to_hmcts_court.mapping
         local-authority-code-to-cafcass-mapping: fpl.local_authority_code_to_cafcass.mapping
         local-authority-code-to-shared-inbox-mapping: fpl.local_authority_code_to_shared_inbox.mapping
+        default-email-mapping: fpl.default_email.mapping
```

Comment:
please make sure to standardise

Reply:
followed the same pattern as above local-authority mapping